code/modules/hydroponics/grown/nymph.dm

```diff
@@ -11,6 +11,45 @@
 	production = 1
 	yield = 1
 	reagents_add = list("plantmatter" = 0.1)
+	var/next_ping_at = 0
+
+// If the nymph pods are fully grown, allow ghosts that click on them to spawn as a nymph (decreases yield by 1 and kills the plant when yield reaches 0)
+/obj/item/seeds/nymph/attack_ghost(mob/dead/observer/O, obj/machinery/hydroponics/H)
+	if(!planted)
```

Comment:
just thought of one idiotic exception to the attack_ghost verb...
Revenants also use this verb at times... so an `istype(O)` check will be needed to prevent revs from using seeds...